### PR TITLE
Fix KeyError

### PR DESCRIPTION
**Pull request by xhl6666**
### Long story short
Reopen PR #192 

### Checks

- [x] I have run `nox` tests and all pipelines have passed.
- [ ] I have wriiten tests according to the code i added, modified or deleted.

### Related issues

### CODE REVIEW

**Comment by nxtlo** [comment]
lgtm :D, But the lint ci is failing, Fixing this should be easy, Just run `nox -s format`, Then make sure it fixed the errors by `nox -s lint`

**Comment by xhl6666** [comment]
Can I add GameMode Enum update commit in this PR instead of open a new PR?

File: aiobungie/internal/factory.py [hunk 1]
```diff
@@ -131,9 +131,8 @@ def deserialize_destiny_membership(
             icon=assets.Image(payload.get("iconPath", "")),
             types=[
                 enums.MembershipType(type_)
-                for type_ in payload["applicableMembershipTypes"]
-                if "applicableMembershipTypes" in payload
-            ],
+                for type_ in payload.get("applicableMembershipTypes", [])
+            ]
```
Comment (nxtlo) [inline aiobungie/internal/factory.py:135]: There's no reason to change this.

**Comment by nxtlo** [comment]
I already added the game modes which will be pushed pretty soon :)

**Comment by nxtlo** [comment]
Closing this due to #197

**Pull request closed without merging**